js/lib/bandstructure.js

```diff
@@ -804,7 +835,7 @@ BandPlot.prototype.updateDosPlot = function (orientation = 'vertical') {
             hidden: false,
             borderWidth: 1,
             data: curve,
-            fill: true,
+            fill: '-1',
```

Comment:
What is this change for?

Reply:
This will fill color between the DOS dataset the dumb dataset (represent y axis).

Reply:
It is still not very clear why the fill can accept a string with a negative integer number. Can you add a comment above to explain why this change is necessary?

Reply:
Thanks for keeping question about this. I read the instruction again and found a mistake. The "-1" fill color to the relative dataset index. I should use 0, which is the absolute dataset index. 

https://www.chartjs.org/docs/latest/charts/area.html

I have corrected this by using: "fill: 0".